PURL-SPECIFICATION.rst

```diff
@@ -1,5 +1,5 @@
-Package URL specification v1.0.X
-================================
+Package URL specification v1.0.X (from qualifiers PR)
```

Comment:
is this change of the title intended for meeging?

Reply:
No; thank you for spotting this @jkowalleck -- it's a temp header I use to check how these .rst pages look on a local test Sphinx RTD before pushing and viewing their GH display.  ;-)  Will remove in the next push along with changes discussed below.

Reply:
Fixed.